templates/patterns/tag/tag.ui_patterns.yml

```diff
@@ -9,7 +9,7 @@ tag:
     tag_type:
       type: "text"
       label: "Type"
-      description: "Type of tag (link, button, removable)"
+      description: "Type of tag (link, removable, display, button (deprecated))"
```

Comment:
Do we use the `button` type anywhere? On EWCMS?

Reply:
I didn't find any usage of 'tag' pattern in EWCMS (and button type neither).